apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -135,6 +135,28 @@ export class AppLivechatBridge extends LivechatBridge {
 		return true;
 	}
 
+	protected async findRoomsByAgentId(agentId: string, appId: string): Promise<ILivechatRoom[]> {
+		this.orch.debugLog(`The App ${appId} is looking for livechat rooms associated with agent ${agentId}`);
+
+		if (!agentId) {
+			return [];
+		}
+
+		const result = await LivechatRooms.findOpenByAgent(agentId).toArray();
```

Comment:
We could use `.map` here directly instead of using `.toArray()`. Wdyt?

Reply:
Is it better to do the `map` instead of `toArray` ? I'm not familiar with the tradeoffs of each option.